service/src/main/java/bio/terra/cbas/dependencies/sam/SamService.java

```diff
@@ -28,4 +36,17 @@ public Result checkHealth() {
       return new Result(false, e.getMessage());
     }
   }
+
+  public UserStatusInfo getUserStatusInfo(String accessToken) {
```

Comment:
@salonishah11 is this an example of a SamService method that should be tested? I'm assuming I would need to mock out the API calls?

Reply:
I think so. You can probably mock out the response from Sam and add unit tests for this function.